src/controller/CHIPDevice.h

```diff
@@ -191,6 +213,8 @@ class DLL_EXPORT Device
 
     NodeId GetDeviceId() const { return mDeviceId; }
 
+    SecureSessionHandle GetSecureSession() const { return mSecureSession; }
```

Comment:
What will this return if `OnNewConnection` has not happened yet?  Will it even be initialized memory?

Reply:
I have removed `GetSecureSession`.